ts/features/itwallet/common/store/reducers/preferences.ts

```diff
@@ -0,0 +1,28 @@
+import { addMonths } from "date-fns";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../../store/actions/types";
+import { itwCloseFeedbackBanner } from "../actions/preferences";
+
+export type ItwPreferencesState = {
+  hideFeedbackBannerUntil?: Date;
```

Comment:
Shouldn't we store only serializable values, ideally?

Reply:
Addressed here: https://github.com/pagopa/io-app/pull/6378/commits/538718682df6589b27cee36dba948792d4fad39e